content/_index.html

```diff
@@ -5,7 +5,7 @@
 
 {{< blocks/cover title="Apache SkyWalking" image_anchor="top" height="full" color="dark" >}}
 <div class="mx-auto">
-    <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://github.com/apache/skywalking/tree/master/docs">
+    <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://skywalking.apache.org/docs/skywalking-showcase/latest/readme/">
```

Comment:
```suggestion
    <a class="btn btn-lg btn-primary mr-3 mb-4" href="/docs/skywalking-showcase/latest/readme/">
```

I think we should use this? Otherwise, 404 shows up on preview page.